core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -79,11 +78,11 @@ final class ShuffleBlockFetcherIterator(
   private[this] val localBlocks = new ArrayBuffer[BlockId]()
 
   /** Remote blocks to fetch, excluding zero-sized blocks. */
-  private[this] val remoteBlocks = new HashSet[BlockId]()
+  private[this] val remoteBlocks = new mutable.HashSet[BlockId]()
```

Comment:
nit: I know this is normal in other scala codebases (and I somewhat prefer it myself because of the easy immutable / mutable confusion), but I think the spark style is actually to just import the fully qualified classes, so i think it should stay as it was before.